trade-finance-manager-ui/templates/utilisation-reports/_macros/report-reconciliation-table.njk

```diff
@@ -2,56 +2,95 @@
-        <td class="govuk-table__cell" data-sort-value="{{ summaryItem.reportedFeesLeftToReconcile }}">
-          {{ summaryItem.reportedFeesLeftToReconcile }}
-        <td class="govuk-table__cell">
-          {% if summaryItem.reportId %}
-            <a
-              class="govuk-link"
-              href="{{ summaryItem.downloadPath }}"
-              aria-label="Download {{ summaryItem.bank.name }} report"
-            >
-              Download
-            </a>
-          {% endif %}
-        </td>
-      </tr>
-    {% endfor %}
-    </tbody>
-  </table>
+      <tbody class="govuk-table__body">
+      {% for summaryItem in reportReconciliationSummary %}
+        {% set checkboxId = ['set-status--bankId-', summaryItem.bank.id] | join %}
+        {% if summaryItem.reportId %}
+          {% set checkboxId = ['set-status--reportId-', summaryItem.reportId] | join %}
+        {% endif %}
+
+        <tr class="govuk-table__row" data-cy="utilisation-report-reconciliation-table-row-bank-{{ summaryItem.bank.id }}">
+          <th
+            scope="row"
+            class="govuk-table__header govuk-!-font-weight-regular"
+            data-sort-value="{{ summaryItem.bank.name }}"
+          >
+            <div class="govuk-checkboxes govuk-checkboxes--small">
+              <div class="govuk-checkboxes__item" data-cy="checkbox--{{ id }}">
+                <input class="govuk-checkboxes__input govuk-!-top-0" id="{{ checkboxId }}" name="{{ checkboxId }}" type="checkbox"/>
+                <label class="govuk-label govuk-checkboxes__label" for="">
+                  <a class="govuk-link govuk-link--no-visited-state" href="#">{{ summaryItem.bank.name }}</a>
+                </label>
+              </div>
+            </div>
+          </th>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.displayStatus }}">
+            {{ reconciliationStatus.render({
+              status: summaryItem.status,
+              displayStatus: summaryItem.displayStatus
+            }) }}
+          </td>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.dateUploaded }}">
+            {{ summaryItem.formattedDateUploaded }}
+          </td>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.totalFeesReported }}">
+            {{ summaryItem.totalFeesReported }}
+          </td>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.reportedFeesLeftToReconcile }}">
+            {{ summaryItem.reportedFeesLeftToReconcile }}
+          <td class="govuk-table__cell">
+            {% if summaryItem.downloadPath %}
+              <a
+                class="govuk-link"
+                href="{{ summaryItem.downloadPath }}"
+                aria-label="Download {{ summaryItem.bank.name }} report"
+              >
+                Download
+              </a>
+            {% endif %}
+          </td>
+        </tr>
+      {% endfor %}
+      </tbody>
+    </table>
+
+    <p class="govuk-body">
```

Comment:
When we add the details of previous reports (see [FN-1051](https://ukef-dtfs.atlassian.net/browse/FN-1051)) will there be a set of buttons for each table, or one set of buttons for all the tables? If the latter then these buttons should go outside of this `report-reconciliation-table` macro.

It's not clear from the designs which of the above we should do, so it's something we'll need to find out (if you haven't already). (@ChristianMcc - you may have thoughts on this)

Reply:
Probably going to have to ask Rich/Andy about this one, in my head it was a form and set of buttons per table. Mostly because I was imagining working on one month at a time and if for some reason one bank was behind by a couple of months it could push the buttons off screen and be hard to see what's going on

Reply:
If the buttons aren't specific to the table being viewed, the checkbox id will need to contain all of the `bankId`, `submissionMonth` and `submissionYear` rather than my current implementation which puts the month and year into the `url`. It is easier to keep the buttons table specific so that we don't need each checkbox to store the month and year in my opinion